source/jormungandr/jormungandr/scenarios/tests/new_default_tests.py

```diff
@@ -120,11 +120,11 @@
     # 10/15/2015 @ 12:08pm (UTC)
     (1444903680, 'best', (0, 8, 9)),
     # J16
-    # 10/15/2015 @ 12:08pm (UTC)
-    (1444903680, 'non_pt_bike', (0,)),
+    # 10/15/2015 @ 12:09pm (UTC)
```

Comment:
I have to modify the arrival time of these 2 journeys so that it won't become the best path